codesets/mlflow/sklearn/train.py

```diff
@@ -57,8 +57,6 @@ def eval_metrics(actual, pred):
         print("  MAE: %s" % mae)
         print("  R2: %s" % r2)
 
-        mlflow.log_param("alpha", alpha)
-        mlflow.log_param("l1_ratio", l1_ratio)
```

Comment:
why removing those?

Reply:
these are not needed because MLFlow logs these automatically. what's worse, in some corner cases, you will run into an error if the values here differ from the ones extracted by MLFlow from your model training (which is what I ran into and why I removed them).